cluster/userdata-master.yaml

```diff
@@ -1126,105 +1126,152 @@ write_files:
+          application: logging-agent
+          version: v0.10
           component: logging
       spec:
         selector:
           matchLabels:
-            application: appdynamics-agent
+            application: logging-agent
         template:
           metadata:
-            name: appdynamics-agent
+            name: logging-agent
             labels:
-              application: appdynamics-agent
-              version: v0.9
+              application: logging-agent
+              version: v0.10
               component: logging
             annotations:
               scheduler.alpha.kubernetes.io/critical-pod: ''
               scheduler.alpha.kubernetes.io/tolerations: '[{"key":"CriticalAddonsOnly", "operator":"Exists"}]'
           spec:
             containers:
             - name: log-watcher
-              image: registry.opensource.zalan.do/eagleeye/k8s-log-watcher:0.9
-              volumeMounts:
-              - name: containerlogs
-                mountPath: /mnt/containers
-                readOnly: true
-              - name: jobfiles
-                mountPath: /mnt/jobs
+              image: registry.opensource.zalan.do/eagleeye/kubernetes-log-watcher:0.11
               env:
               - name: CLUSTER_NODE_NAME
                 valueFrom:
                   fieldRef:
                     fieldPath: spec.nodeName
-              - name: WATCHER_KUBE_URL
-                value: http://127.0.0.1:8001
+
+              - name: WATCHER_KUBERNETES_UPDATE_CERTIFICATES
+                value: "true"
               - name: WATCHER_DEBUG
-                value: "1"
+                value: "true"
+              - name: WATCHER_CLUSTER_ID
+                value: "{{CLUSTER_ID}}"
+
+              - name: WATCHER_AGENTS
+                value: scalyr
+              - name: WATCHER_SCALYR_API_KEY
+                value: "{{SCALYR_ACCESS_KEY}}"
+              - name: WATCHER_SCALYR_DEST_PATH
+                value: /mnt/scalyr-logs
+              - name: WATCHER_SCALYR_CONFIG_PATH
+                value: /mnt/scalyr-config/agent.json
+              - name: WATCHER_SCALYR_JOURNALD
+                value: "true"
+
               resources:
                 limits:
                   cpu: 200m
                   memory: 200Mi
                 requests:
                   cpu: 100m
                   memory: 80Mi
-            - name: appdynamics-agent
-              image: pierone.stups.zalan.do/monitoring-appdynamics/appdynamics-machine-agent:0.3
+
               volumeMounts:
               - name: containerlogs
                 mountPath: /mnt/containers
                 readOnly: true
-              - name: jobfiles
-                mountPath: /appd/agent/monitors/analytics-agent/conf/job
+              - name: scalyr-logs
+                mountPath: /mnt/scalyr-logs
+                readOnly: false
+              - name: scalyr-config
+                mountPath: /mnt/scalyr-config
+                readOnly: false
```

Comment:
Yes, the ``log-watcher`` updates the scalyr-config ``agent.json`` which is then consumed and detected by ``scalyr-agent`` container below.